test/boost/cdc_test.cc

```diff
@@ -48,87 +48,9 @@ static cql_test_config mk_cdc_test_config() {
 };
 
 namespace cdc {
-api::timestamp_type find_timestamp(const schema&, const mutation&);
 utils::UUID generate_timeuuid(api::timestamp_type);
 }
 
-SEASTAR_THREAD_TEST_CASE(test_find_mutation_timestamp) {
```

Comment:
Why is this deleted?

Reply:
there is no `find_timestamp` function anymore

Reply:
Yes, but can't this test somehow be modified to test the new `extract_changes()` instead?

Reply:
Yes, I'm planning to add a new test. It will test much more than just whether timestamps are extracted properly.